src/common/imageio_rawspeed.cc

```diff
@@ -257,6 +257,11 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
     for(int i = 0; i < 4; i++)
       img->wb_coeffs[i] = r->metadata.wbCoeffs[i];
 
+    // Grab the adobe coeff
+    for(int k = 0; k < 4; k++)
+      for(int i = 0; i < 3; i++)
+        img->adobe_XYZ_to_CAM[k][i] = r->metadata.colorMatrix[k*3+i] / 10000.0;
```

Comment:
Should maybe be `10000.0f` to avoid implicit casts...
 
Perhaps we should define the 10000 denominator macro as it is used in a few places (like `imageio_dng.h` above)?

Reply:
Done.